src/docx/assets/specjalne/customXml/item1.xml

```diff
@@ -0,0 +1,3 @@
+<b:Sources SelectedStyle="\APA.XSL" StyleName="APA Fifth Edition" 

```

Comment:
Don't know if such multiline will be properly rendered in MS Word? Don't really know if XML spec allows splitting tags in lines.

Reply:
It renders properly, so far. Also in MS Word.